flocker_bb/builders/flocker.py

```diff
@@ -395,6 +396,12 @@ def makeOmnibusFactory(distribution, triggerSchedulers=()):
             "setup.py", "sdist",
             ],
         haltOnFailure=True))
+    if distribution == 'fedora-20':
```

Comment:
If we are going to share the sdist between builds, it probably makes sense to have a builder that just builds that, and then triggers everything that needs it. That is, create a new builder that builds the sdist (and wheel?) and uploads them and then triggers all the omnibus builders and the new builder.

On the other hand, building sdists is cheep, so perhaps the new builder can just build the sdist itself and upload it, and the omnibus builders can just build their own.